io.openems.edge.io.shelly/bnd.bnd

```diff
@@ -5,12 +5,13 @@ Bundle-Version: 1.0.0.${tstamp}
 
 -buildpath: \
 	${buildpath},\
+	com.ghgande.j2mod,\
 	io.openems.common,\
 	io.openems.edge.bridge.http,\
 	io.openems.edge.common,\
 	io.openems.edge.io.api,\
 	io.openems.edge.meter.api,\
 	io.openems.edge.timedata.api,\
-
+	io.openems.edge.bridge.modbus
```

Comment:
This will break `prepare-commit.sh` script. Please always run before opening a PR - see https://openems.github.io/openems.io/openems/latest/contribute/coding-guidelines.html#_openems_edge_backend